Info-Orbs/src/main.cpp

```diff
@@ -7,7 +7,7 @@
 #include <Button.h>
 #include <globalTime.h>
 #include <config.h>
-#include <widgets/screenWidgets/stockScreenWidget.h>
+#include <widgets/stockScreenWidget.h>
```

Comment:
I think the widget should simply be `stockWidget` now as the Screen part is meaningless.

Reply:
I was thinking about that as well.... You may have had it commented out with that names and I just ran with it. I agree removing "screen" from the name.